.tours/2---object agnostic design.tour

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "2 - Object-Agnostic Design",
+    "steps": [
+        {
+            "file": "es-base-code/main/default/objects/Customer_Fields__mdt/fields/Sobject_Type__c.field-meta.xml",
+            "description": "The app uses Custom Metadata Customer Fields to allow administrators to configure Lightning Web Components and Flow Screen Components to work with any object",
+            "line": 7
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/customerList/customerList.html",
+            "description": "The customerList and customerTile Lightning Web Components can display information from any object as long as the object names and fields are in the Custom Metadata Customer Fields",
```

Comment:
May be go a lil deep to the exact line in the component/JS file where this is used.

Reply:
done!